### ISSUE TITLE
We need better species images for at least the "main" species

### ISSUE THREAD

**Issue by sammyjava** [legumeinfo/jekyll-legumeinfo#16]
I've grabbed some images off Google Images, but I think someone should put some quality time into more consistent images for at least the main species (glyma.jpg, glyso.jpg, arahy.jpg, etc.). Feel free to replace if you've got a better one. The width should be 250 to 400 pixels.

**Comment by cann0010** [comment]
Will work on this. For the Glycine species, we have decent images, but they need some work (combining, cropping, sizing).
How to share these? Data Store private?

**Comment by cann0010** [comment]
Images for Glycine are now on lis-stage, at /usr/local/www/data/private/Glycine/images/
@sammyjava - you'll want to grab the ones in pixels_400/
(Leaving the issue open, since I'm sure there are others that we'll need)

**Comment by adf-ncgr** [comment]
it might be good if somewhere we could indicate the attribution/licensing of the images, if only as a way of forcing us to try to ensure we are not violating any copyrights with the images we grab. I wouldn't normally think of such things, but I vaguely recall an incident with a project website NCGR had set up once where we got hit by a copyright troll due to some negligence in this regard.

**Comment by cann0010** [comment]
Agree re attribution. @sammyjava - what do you think about new outer-level tags in the description file:
```
imageName: 
attribution: 
```

**Comment by sammyjava** [comment]
Let's just use CC-licensed images throughout. I grabbed some from Wikimedia Commons, which are all CC. No one will sue us for having CC licensed images on our site. Again, I'd prefer that we not use the description files for web-specific instructions, and images seem to me to be more of a website-related thing than a taxa/DS information thing.

Let's focus on more important things so we can get the site live in February. I'll snag those Glycine images, thanks, Steven!

**Comment by sammyjava** [comment]
Those are really nice images, @cann0010 ! http://dev.lis.ncgr.org:50030/taxa/glycine/

**Comment by cann0010** [comment]
Thanks. I put them together this morning and tweaked them a bit. They came via Jeff Doyle a few years ago. I need to ask him for details.
I don't mind plowing ahead for now, but I think we're better off formally "futuring" the issue and coming back to it, than saying that we'll just rely on CC-licensed images. A lot of the images will be coming from research friends (the Bertiolis, Jeff Doyle, etc.) They're not going to sue us; but we should still provide attributions. Someone (maybe future-us) is going to say: Nice picture. Where did you get it?

**Comment by sammyjava** [comment]
Fair enough, let's focus on the actual task of generating images under this issue, and I'll create a separate issue re. changing the DS/code to deal with attribution.

**Comment by sammyjava** [comment]
Here's a list of images that are missing that I think we might like to have:

- [ ] Phaseolus acutifolius
- [ ] Aeschynomene evenia
- [ ] the "special" genera that are missing entirely
